mev_inspect/schemas/classified_traces.py

```diff
@@ -24,32 +24,53 @@ class Protocol(Enum):
     zerox = "0x"
 
 
-class ClassifiedTrace(BaseModel):
+class ClassifiedTrace(Trace):
+
     transaction_hash: str
     block_number: int
-    trace_type: TraceType
     trace_address: List[int]
     classification: Classification
-    protocol: Optional[Protocol]
-    abi_name: Optional[str]
-    function_name: Optional[str]
-    function_signature: Optional[str]
-    inputs: Optional[Dict[str, Any]]
+
+    # Optional
+    error: Optional[str]
     to_address: Optional[str]
     from_address: Optional[str]
     gas: Optional[int]
     value: Optional[int]
     gas_used: Optional[int]
-    error: Optional[str]
+    protocol: Optional[Protocol]
+    function_name: Optional[str]
+    function_signature: Optional[str]
+    inputs: Optional[Dict[str, Any]]
```

Comment:
is it `Any`? I thought you learned it was bytes always

Reply:
It can actually also be a tuple of bytes